modules/serverpod_auth/serverpod_auth_email_flutter/lib/src/signin_dialog.dart

```diff
@@ -533,7 +550,10 @@ class SignInWithEmailDialogState extends State<SignInWithEmailDialog> {
     _resetIssues();
     if (_passwordController.text.length < widget.minPasswordLength) {
       setState(() {
-        _passwordIssue = 'Min ${widget.minPasswordLength} characters';
+        _passwordIssue = widget.labels?.minimum != null &&
+                widget.labels?.characters != null
+            ? '${widget.labels?.minimum} ${widget.minPasswordLength} ${widget.labels?.characters}'
```

Comment:
Comment: The problem with this type of contamination is that it may not work for all languages. When doing plurality and numbers i18n packages normally have a callback of some kind and give you the number so you can template the string however you want. 

Suggestion: I think it is likely a better idea to make this a callback that takes the minPasswordLength and expects the full string back. Instead of "minimum" and "characters".